stock_quant_reservation_info/models/stock_quant.py

```diff
@@ -9,7 +9,7 @@ class StockQuant(models.Model):
     def action_reserved_moves(self):
         self.ensure_one()
         action = {
-            "name": _("Reserved moves for: " + self.product_id.name),
+            "name": _("Reserved Moves for: %s") % self.product_id.name,
```

Comment:
`_("Reserved Moves for: %(product_name)s", product_name=self.product_id.name)` is better

Reply:
Changes are done